clang/test/CIR/IR/getmember.cir

```diff
@@ -14,11 +12,4 @@ module  {
     %0 = cir.get_member %arg0[1] {name = "test"} : !cir.ptr<!ty_Struct> -> !cir.ptr<!u32i>
     cir.return
   }
-
-  // FIXME: remove bypass once codegen for CIR class records is patched.
-  cir.func @shouldBypassMemberTypeCheckForClassRecords(%arg0 : !cir.ptr<!ty_Class>) {
```

Comment:
So turns out we had a test for this, nice. Does this snippet now fails after this PR? Code looks legit so I wonder if it should just become a regular test (not needing extra workaround) now that the "bypass" was lifted.

Reply:
Yes, this snippet would fail the verifier after this PR. I'm not sure what the original intent of this CIR was. The verifier fails because there's an extra level of indirection in the return value.

Reply:
I can't remember either, well, let's see if we get a verification error once this propagates to our internal CI, I can come back and revert worse case.